### PR TITLE
Release 5.0.1

### PR DESCRIPTION
**Pull request by jacadzaca**
Trying out the new release process as asked of me in #467, please merge #469 first.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 =========
 
-5.0.1 (unreleased)
+5.0.2 (unreleased)
 ------------------
 
 Minor changes:
 
-- fixed setuptools deprecation warnings [mgorny]
+- ...
```
Comment (niccokunzmann) [inline CHANGES.rst:9]: It is good to not have this in the tag as it turns up on PyPI.

**Review by niccokunzmann — APPROVED** [review]

**Comment by niccokunzmann** [comment]
Cool! I am awaiting your tag :)

**Comment by jacadzaca** [comment]
Did it work?

**Comment by niccokunzmann** [comment]
See for yourself... yes!

**Comment by mauritsvanrees** [comment]
Thanks for the release.
One small thing went wrong though. The 5.0.1 release already contains the dummy changelog entries for "5.0.2 (unreleased)". See https://github.com/collective/icalendar/blob/v5.0.1/CHANGES.rst

Previously I used the `fullrelease` command from `zest.releaser` to do the releases, and that would fix this. See the [overview in the documentation](https://zestreleaser.readthedocs.io/en/latest/) for what it does. But I don't know if this tool would be a good match with the current release process, with making a release, approving it, and letting GitHub push the release to PyPI. It *should* though: me and my brother made this tool to automate the boring bits of doing the release.

**Comment by niccokunzmann** [comment]
I think, it is worth turning your suggestions into an issue. I adjusted the release process documentation to not include an empty changelog.

**Pull request merged**